### PR TITLE
Add z field

### PR DESCRIPTION
**Pull request by mckabuza**

### CODE REVIEW

**Comment by mckabuza** [comment]
Pull request to add zField to the zCluster code.

File: bin/zField [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+        zDebias=0.02    # From testing against spec-zs (don't know yet why there is a mean offset)
+        retrieverOptions={'maxMagError': maxMagError, 'bricksTab': bricksTab, 'DR8Tab': DR8Tab, 'altCacheDir': "DECaLSCache"}
+    elif database == 'CFHTDeep':
+        retriever=catalogRetriever.CFHTDeepRetriever
+    elif database == 'CFHTWide':
+        retriever=catalogRetriever.CFHTWideRetriever
+        zPriorMin=0.05
+        zPriorMax=1.5
+        
+    if passbandSet == 'SDSS+Ks':
```
Comment (mattyowl) [inline bin/zField]: Instead of this block here, get the names of the bands from PhotoRedshiftEngine.bands.

File: bin/zField [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+                arr.append(gobj[key])
+            except:
+                arr.append(99)
+        tab.add_column(atpy.Column(np.array(arr), key))
+    tab.write(outFileName, overwrite = True)
+        
+    if writeDensityMaps == True:
+        dMapDict = clusterFinding.makeDensityMap(RADeg, decDeg, galaxyCatalog, args.z, args.dz, rMaxMpc = 1.5)
+        astImages.saveFITS(args.outDir+os.path.sep+"densityMap_%s.fits" % (database.replace(" ", "_")),
+                           dMapDict['map'], dMapDict['wcs'])s
```
Comment (mattyowl) [inline bin/zField]: Syntax error - you would see this just by trying to run zField -h

**Review by mattyowl — CHANGES_REQUESTED** [review]
Not tried this review thing before - see if you can see/address the comments. If not, let me know

**Review by mattyowl — APPROVED** [review]
Thanks for sorting this

**Pull request merged**